### PR TITLE
create crowdsec group for metabase and crowdsec.db

### PR DESCRIPTION
**Pull request by AlteredCoder**
- Fix in wizard that display the warning message when providing `--force`
- Fix docker metabase image deletion
- Create a crowdsec group for docker and make it owner of the crowdsec.db group
- change `cscli dashboard remove --force` by `cscli dashboard remove --purge` (since this delete the metabase docker image)

### CODE REVIEW

**Review by registergoofy — APPROVED** [review]
lgtm

File: cmd/crowdsec-cli/dashboard.go [hunk 1]
```diff
@@ -72,7 +76,48 @@ cscli dashboard setup -l 0.0.0.0 -p 443 --password <password>
...
-			mb, err := metabase.SetupMetabase(csConfig.API.Server.DbConfig, metabaseListenAddress, metabaseListenPort, metabaseUser, metabasePassword, metabaseDbPath)
+			var answer bool
+			groupExist := false
+			dockerGroup, err := user.LookupGroup(crowdsecGroup)
+			if err == nil {
+				groupExist = true
+			}
+			if !forceYes && !groupExist {
+				prompt := &survey.Confirm{
+					Message: fmt.Sprintf("We need to add a new group called '%s' to the system, is it ok for you ?", crowdsecGroup),
```
Comment (buixor) [inline cmd/crowdsec-cli/dashboard.go:87]: we need a more explicit message, ie. "for metabase docker to be able to access SQLite file, we need to add a group called ..."

File: cmd/crowdsec-cli/dashboard.go [hunk 2]
```diff
@@ -72,7 +76,48 @@ cscli dashboard setup -l 0.0.0.0 -p 443 --password <password>
...
+					Default: true,
+				}
+				if err := survey.AskOne(prompt, &answer); err != nil {
+					log.Fatalf("unable to ask to force: %s", err)
+				}
+			}
+			if !answer && !forceYes {
```
Comment (buixor) [inline cmd/crowdsec-cli/dashboard.go:94]: I don't think we want to fatal here (ie. user can do it on its own), but rather a strong warning

File: cmd/crowdsec-cli/dashboard.go [hunk 3]
```diff
@@ -149,6 +195,18 @@ cscli dashboard remove --force
...
 			}
+			dockerGroup, err := user.LookupGroup(crowdsecGroup)
+			if err == nil { // if group exist, remove it
+				groupDelCmd, err := exec.LookPath("groupdel")
+				if err != nil {
+					log.Fatalf("unable to find 'groupdel' command, can't continue")
+				}
+
+				groupDel := &exec.Cmd{Path: groupDelCmd, Args: []string{groupDelCmd, crowdsecGroup}}
+				if err := groupDel.Run(); err != nil {
```
Comment (buixor) [inline cmd/crowdsec-cli/dashboard.go:206]: I don't think we want to fatal if the group wasn't deleted

**Review by buixor — CHANGES_REQUESTED** [review]

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**